### PR TITLE
chore(devenv): fix for Safari

### PR DESCRIPTION
**Pull request by asudoh**
🙏 CC @tw15egan 

#### Changelog

**Changed**

- Changed the way to detect if the dev env UI is ready for bootstrap.

#### Testing / Reviewing

Testing should make sure our dev env is not broken.

### CODE REVIEW

File: demo/js/components/boot-nav.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ import RootPage from './RootPage';
  * @private
  */
 const pollForBrowserSync = callback => {
-  const bsScriptTag = document.getElementById('__bs_script__');
+  const bsScriptTag = Array.prototype.find.call(document.querySelectorAll('script'), elem =>
```
Comment (joshblack) [inline demo/js/components/boot-nav.js:13]: Do we polyfill for `find` already?

**Review by tw15egan — APPROVED** [review]
Works great for me! Thanks for fixing this so quickly 😄

**Review by alisonjoseph — APPROVED** [review]
works for me, as long as Josh's question is addressed

File: demo/js/components/boot-nav.js (hunk 1 above)
Comment (asudoh) [reply demo/js/components/boot-nav.js:13]: @joshblack Good question, and it's yes: https://github.com/IBM/carbon-components/blob/bed6f1eebb31e8a3e1e44cab0297c2c420a2dd09/demo/js/components/boot-nav.js#L1

**Pull request merged**